modules/appeals_api/app/workers/appeals_api/daily_error_report.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module AppealsApi
+  class DailyErrorReport
+    include Sidekiq::Worker
+
+    def perform
+      DailyErrorReportMailer.build.deliver_now if Settings.modules_appeals_api.reports.daily_error.enabled
```

Comment:
This will technically still fail if the `reports` namespace is missing. Should be ok with either doing safe method operators (x&.y) or digging as you did in DecisionReviewMailer.